index.html

```diff
@@ -2859,7 +2868,16 @@ <h3>Prefer Single-Use Credentials</h3>
 to operate within.
         </p>
       </section>
-
+        <section>
+            <h3>Avoid Disclosing Credential Identifier</h3>
+            <p>
+                Disclosing the credential identifier leads to situations where multiple
+verifiers or an issuer and a verifier can collude to correlate the holder. The holder should
+use credential schemes which allow to hide the identifier during presentation. Such schemes
+expect the holder to generate the identifier and may even allow to hide the identifier from
+the issuer, still keeping the identifier embedded (and signed) in the credential
```

Comment:
`issuer, still` => `issuer, whilst still` ... also missing a period at the end.

Reply:
Changed